client/images/p5js-logo-small.svg

```diff
@@ -1,20 +1,20 @@
 <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="56" height="42" viewBox="0 0 56 42">
     <defs>
-        <path id="a" d="M0 0h36v36H0z"/>
+        <path id="a" d="M0 0h36v36H0z" />
         <path id="b" d="M0 0h36v36H0z"/>
     </defs>
-    <g fill="none" fill-rule="evenodd">
-        <path fill="#ED225D" d="M0 0h56v42H0z"/>
+    <g fill="none" fill-rule="evenodd" class="main__svg">
+        <path class="main__svg-part" d="M0 0h56v42H0z"/>
```

Comment:
you don't need to change the SVG element—you can apply a class to the element that contains the svg element, and then target it by doing
```scss
.my_class g {
  @include themify() {
      fill: getThemifyVariable('logo-color');
    }
}
```

Reply:
@catarak This doesn't work.

Reply:
Because I don't want to change the property of all` <g>` tags. and there are `<g>` nested inside each other `<g>`. Is there any other approach?